app/src/components/market/sections/market_create/steps/outcomes/index.tsx

```diff
@@ -266,11 +268,15 @@ const Outcomes = (props: Props) => {
             disabled={disableManualProbabilities}
             formField={
               <Textfield
-                onChange={e => setNewOutcomeProbability(Number(e.target.value))}
+                min="0.00"
+                onChange={e => {
+                  setNewOutcomeProbability(e.target.value)
+                }}
                 onKeyUp={e => {
                   onPressEnter(e)
                 }}
                 placeholder="0.00"
+                step="0.01"
```

Comment:
What does `step` do?

Reply:
well just extra measure that number can be only in two decimals...

Reply:
This may work for stablecoins but for example Bitcoin (WBTC) could use lower then 0.01 inputs.

Reply:
ok i will delete that